src/libYARP_OS/include/yarp/os/Random.h

```diff
@@ -8,7 +8,7 @@
 #define YARP_OS_RANDOM_H
 
 #include <yarp/os/api.h>
-
+#include <random>
```

Comment:
Please move this commit on devel

Reply:
This commit is also changing a lot the semantics of the call. It deserves a separate PR and  a proper ChangeLog line.